androidApp/home/src/main/java/tech/takahana/iconwallpaper/android/home/ui/util/DrawScopeUtils.kt

```diff
@@ -0,0 +1,79 @@
+package tech.takahana.iconwallpaper.android.home.ui.util
+
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.drawscope.DrawScope
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.IntSize
+
+object DrawScopeUtils {
+    fun DrawScope.drawPattern(
+        image: ImageBitmap,
+        backGroundColor: Color,
```

Comment:
ここも `backgroundColor` でいいかも👀

```suggestion
        backgroundColor: Color,
```

Reply:
こちらのコミットで修正しました！

117b9d3f062e6f02277400f5259929fb99836519